WordPress/src/main/res/layout/progress_layout.xml

```diff
@@ -9,13 +9,11 @@
     android:gravity="center"
     android:orientation="vertical">
 
-    <TextView
+    <TextSwitcher
         android:id="@+id/progress_text"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
```

Comment:
`match_parent` improves flickering of layout since different texts have different lengths and it creates an odd effect with `TextSwitcher` (it uses 2 `TextView` instance under the hood)